database/seeders/DatabaseSeeder.php

```diff
@@ -13,7 +14,14 @@ class DatabaseSeeder extends Seeder
      */
     public function run()
     {
-        $this->call(UserSeeder::class);
+        $this->call([
```

Comment:
@thanhdat02112001 
- Seed 1000 user
  - 100 user chưa verify
  - 400 user đã verify email, chưa thanh toán
  - 500 user đã verifyemail, đã thanh toán Stripe
- Mỗi user random từ 1 ~ 3 store: tùy theo loại user
- Mỗi store random từ 10 ~ 100 product: tùy theo loại user